store/postgres/src/entities.rs

```diff
@@ -309,7 +309,7 @@ pub(crate) fn make_storage_cache() -> StorageCache {
 /// been migrated
 #[derive(Constructor)]
 pub(crate) struct Connection<'a> {
-    conn: MaybeOwned<'a, PooledConnection<ConnectionManager<PgConnection>>>,
+    pub conn: MaybeOwned<'a, PooledConnection<ConnectionManager<PgConnection>>>,
```

Comment:
`pub(crate)`

Reply:
The struct us already `pub(crate)`, and the module is not exported outside the crate - is the `pub(crate)` still necessary here then?

Reply:
Ok this can be kept as it is.